install-dev/upgrade/sql/1.7.3.0.sql

```diff
@@ -3,3 +3,5 @@ SET NAMES 'utf8';
 
 UPDATE `PREFIX_tab` SET `position` = 0 WHERE `class_name` = 'AdminZones' AND `position` = '1';
 UPDATE `PREFIX_tab` SET `position` = 1 WHERE `class_name` = 'AdminCountries' AND `position` = '0';
+
+ALTER TABLE `PREFIX_feature_product` DROP PRIMARY KEY ,ADD PRIMARY KEY (`id_feature`, `id_product`, `id_feature_value`);
```

Comment:
Not a big deal, but the comma in `DROP PRIMARY KEY ,ADD PRIMARY KEY` is misaligned